src/tools/melt/melt.cs

```diff
@@ -252,6 +253,52 @@ private void MeltModule()
         }
 
         /// <summary>
+        /// Extract binary data from tables with a Name and Data column in them.
+        /// </summary>
+        /// <param name="inputPdb">A reference to a <see cref="Pdb"/> as output.  Paths (Data properties) will be modified in this object.</param>
+        /// <param name="package">The installer database to rip from.</param>
+        /// <param name="exportPath">The full path where files will be exported to.</param>
+        /// <param name="tableName">The name of the table to export.</param>
+        private void MeltBinaryTable(ref Pdb inputPdb, InstallPackage package, string exportPath, string tableName)
+        {
+            if(string.IsNullOrEmpty(tableName))
```

Comment:
DTF's different but WiX style is a space between keyword and opening parens. (Not for function invocations, though.)